docs/en/install-upgrade/installing-stack-demo-self.asciidoc

```diff
@@ -0,0 +1,713 @@
+
+. Save your changes and close the editor.
+
+[discrete]
+[[install-stack-self-elasticsearch-start]]
+== Step 3: Start {es}
+
+. Now, it's time to start the {es} service:
++
+["source","sh",subs="attributes"]
+----
+sudo systemctl start elasticsearch.service
+----
++
+If you need to, you can stop the service by running `sudo systemctl stop elasticsearch.service`.
+
+. Make sure that {es} is running properly.
++
+["source","sh",subs="attributes"]
+----
+sudo curl --cacert /etc/elasticsearch/certs/http_ca.crt -u elastic:$ELASTIC_PASSWORD https://localhost:9200
+----
++
+In the command, replace `$ELASTIC_PASSWORD` with the `elastic` superuser password that you copied from the install command output.
++
+If all is well, the command returns a response like this:
++
+["source","js",subs="attributes,callouts"]
+----
+{
+  "name" : "Cp9oae6",
+  "cluster_name" : "elasticsearch",
+  "cluster_uuid" : "AT69_C_DTp-1qgIJlatQqA",
+  "version" : {
+    "number" : "{version_qualified}",
+    "build_type" : "{build_type}",
+    "build_hash" : "f27399d",
+    "build_flavor" : "default",
+    "build_date" : "2016-03-30T09:51:41.449Z",
+    "build_snapshot" : false,
+    "lucene_version" : "{lucene_version}",
+    "minimum_wire_compatibility_version" : "1.2.3",
+    "minimum_index_compatibility_version" : "1.2.3"
+  },
+  "tagline" : "You Know, for Search"
+}
+----
+
+. Finally, check the status of {es}:
++
+[source,"shell"]
+----
+sudo systemctl status elasticsearch
+----
++
+As with the previous `curl` command, the output should confirm that {es} started successfully. Type `q` to exit from the `status` command results.
+
+[discrete]
+[[install-stack-self-elasticsearch-second]]
+== Step 4: Set up a second {es} node
+
+To set up a second {es} node, the initial steps are similar to those that you followed for <<install-stack-self-elasticsearch-first>>.
+
+. Log in to the host where you'd like to set up your second {es} instance.
+
+. Create a working directory for the installation package:
++
+["source","shell"]
+----
+mkdir elastic-install-files
+----
+
+. Change into the new directory:
++
+["source","shell"]
+----
+cd elastic-install-files
+----
+
+. Download the {es} RPM and checksum file:
++
+["source","sh",subs="attributes"]
+----
+wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-{version}-x86_64.rpm
+wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-{version}-x86_64.rpm.sha512
+----
+
+. Check the SHA of the downloaded RPM:
++
+["source","sh",subs="attributes"]
+----
+shasum -a 512 -c elasticsearch-{version}-x86_64.rpm.sha512
+----
+
+. Run the {es} install command:
++
+["source","sh",subs="attributes"]
+----
+sudo rpm --install elasticsearch-{version}-x86_64.rpm
+----
++
+Unlike the setup for the first {es} node, in this case you don't need to copy the output of the install command, since these settings will be updated in a later step.
+
+. Enable {es} to run as a service:
```

Comment:
Can we make sure that we also say not to start the service yet (it's important)?

Reply:
Added:
![Screenshot 2024-01-11 at 2 55 19 PM](https://github.com/elastic/stack-docs/assets/41695641/d46072e0-69c7-425d-8a4b-83195eeff842)